transcom/mymove
# B 18679 INT - Add additional information to search results page for QAE/CSR

## [B-18679](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A905412&RoomContext=TeamRoom%3A855664&concept=TeamRoom)

## Summary
This story adds Origin GBLOC, Scheduled Pickup Date, Destination GBLOC, and Scheduled Delivery Date columns to the QAE/CSR user's search results table.

Is there anything you would like reviewers to give additional scrutiny?
Since each shipment in a move can have its own scheduled dates this uses the date from the first shipment.

React-table recently switched to v8 and no longer hosts docs for v7 (what we currently use). I found [React-Table v7 Docs here](https://react-table-v7-docs.netlify.app/docs)

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Ensure there are several moves available to view as a QAE/CSR user (I assume the e2e_populate that I can't yet run provides those)
2. Login as a QAE/CSR user
3. Search for moves via any of the three search types
4. Ensure all four new columns are shown and display correct data. The date columns should be empty unless there is a scheduled date.
5. Ensure sorting and filtering function correctly where applicable.


### Frontend


### Backend
